[Blazor] Data binding - "Expressions" - alternative intro proposal (#32535)
dotnet/AspNetCore.Docs: aspnetcore/blazor/components/data-binding.md

## Bound field or property expression tree

> To facilitate deeper interactions with the bindings, Blazor allows the capture of the [expression tree](/dotnet/csharp/advanced-topics/expression-trees/) of a bound field or property. This is achieved by defining a property named in accordance with the field or property it relates to, suffixed by `Expression`. Thus, for any given parameter or field named `{PARAMETER OR FIELD NAME}`, the corresponding expression tree property should be named `{PARAMETER OR FIELD NAME}Expression`.

Reviewer: ```suggestion
To facilitate deeper interactions with a binding, Blazor allows you to capture of the [expression tree](/dotnet/csharp/advanced-topics/expression-trees/) of a bound field or property. This is achieved by defining a property with the field or property name suffixed with `Expression`. For any given field or property named `{FIELD OR PROPERTY NAME}`, the corresponding expression tree property is named `{FIELD OR PROPERTY NAME}Expression`.
```
